Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -689,3 +689,18 @@
   categories:
   - UplinkDeception
+
+- type: listing
+  id: NukieUplinkMysteriousScroll
+  name: uplink-sleeping-carp-name
+  description: uplink-sleeping-carp-desc
+  productEntity: SleepingCarpScroll
+  cost:
+    Telecrystal: 250 #nukie can use it by all team
+  categories:
+  - UplinkWeaponry
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)